data/lang/module-advice/en.json

```diff
@@ -8,8 +8,8 @@
     "message": "Know when to leave town"
   },
   "ADVICE_1_BODYTEXT": {
-    "description": "Point made is: the real illegal goods traders are fix to each station, but police's fake illegal goods traders will be random every time.",
-    "message": "As you return to a station, be weary of those alternative market places you do not recognize, and you will stay out of trouble with the law."
+    "description": "Point made is: the real illegal goods traders are fixed to each station, but police's fake illegal goods traders will be random every time.",
```

Comment:
I'm not convinced that 'fixed' is preferred over 'fix', as it not referring to something that is "put in order"/fixed, but rather, something that is constant. Native speakers can chime in on this if I'm wrong.

Reply:
Native speaker here (although school is a distant memory 😄)

The two definitions of the verb "to fix" still use the same conjugation and past participle (I think that's the right term for its use here). Replace with the verb "paint" and see what you get:

"The products are paint to each wall" / "The products are painted to each wall"

Reply:
"Fixed" in this instance is a shorter and more common form of the verb "affixed", as in "to pin/nail/glue in place" or "to permanently attach". "Fix" in this instance doesn't have any semantic meaning other than a mis-conjugation of "fixed".

Reply:
So, e.g. "The North star is affixed / fixed in the northern sky"?

Reply:
Yes, "The North Star is fixed at a constant heading in the northern sky" would be valid - here "fixed" means it's in a constant position. It's a little weird because "affixed" is an action verb and implies that "we" (for some definition of we) *did* the affixing, but that's more of an English idiosyncrasy, and likely why the "fixed" form tends to be more popular - because it doesn't imply the subject of the verb so strongly.